src/screens/ExplainSheet.js

```diff
@@ -384,6 +384,28 @@ export const explainers = (params, colors) => ({
       </Row>
     ),
   },
+  obtainL2Assets: {
+    buttonText: 'Go to the Hop Bridge 􀮶',
+    buttonColor: colors?.networkColors[params?.network],
+    secondaryButtonText: lang.t('explain.learn_more'),
+    logo: (
+      <ChainBadge
+        assetType={params?.network}
+        marginBottom={8}
+        position="relative"
+        size="large"
+      />
+    ),
+    readMoreLink:
+      'https://learn.rainbow.me/a-beginners-guide-to-layer-2-networks',
```

Comment:
This may need to be replaced with a 'forthcoming knowledge base article' url

Reply:
can you add a ticket 4 this?